<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNote</class>
 <widget class="QDialog" name="AddNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="14" column="0" colspan="2">
      <widget class="Input" name="birthdate_input"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="first_name_label">
       <property name="text">
        <string>Prénom: </string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="Input" name="name_input"/>
     </item>
     <item row="19" column="0" colspan="2">
      <widget class="Input" name="mail_input"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="birthdate_label">
       <property name="text">
        <string>Date de naissance: </string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="2">
      <widget class="QLabel" name="promo_label">
       <property name="text">
        <string>Promo:</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="QPushButton" name="accept_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nickname_label">
       <property name="text">
        <string>Surnom: </string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="phone_label">
       <property name="text">
        <string>Téléphone:</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="photo_label">
       <property name="text">
        <string>Photo:</string>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0" colspan="2">
      <widget class="QPushButton" name="photo_button">
       <property name="text">
        <string>Ajouter une photo</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="mail_label">
       <property name="text">
        <string>Mail:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="2">
      <widget class="QComboBox" name="promo_input">
       <item>
        <property name="text">
         <string>1A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Externe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ESIAB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prof</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ancien</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Input" name="nickname_input"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="Input" name="first_name_input"/>
     </item>
     <item row="21" column="0" colspan="2">
      <widget class="Input" name="phone_input"/>
     </item>
     <item row="24" column="0" colspan="2">
      <widget class="QLabel" name="photo">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QLineEdit</extends>
   <header>gui.input</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nickname_input</tabstop>
  <tabstop>first_name_input</tabstop>
  <tabstop>name_input</tabstop>
  <tabstop>birthdate_input</tabstop>
  <tabstop>promo_input</tabstop>
  <tabstop>mail_input</tabstop>
  <tabstop>phone_input</tabstop>
  <tabstop>accept_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>photo_button</sender>
   <signal>clicked()</signal>
   <receiver>AddNote</receiver>
   <slot>add_photo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accept_button</sender>
   <signal>clicked()</signal>
   <receiver>AddNote</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>AddNote</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_photo()</slot>
 </slots>
</ui>
